spec/sim/0011-split-comp-rewards/hypothetical_mainnet_upgrade.scen

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env -S yarn repl -s
```

Comment:
does this pass? I'm getting a revert when I run it
```
Command: From TokenHolder (CToken cMKR Mint 1000e18)
Action: CToken cMKR: TokenHolder (0x05e7...) mints 1e+21
Error: Success invariant broken! Expected successful execution, but had error Invokation<"mint(mintAmount=1000000000000000000000)", tx=, value=0, error=InvokationError<err=RuntimeError: VM Exception while processing transaction: revert re-entered>, failures=> processing From TokenHolder (CToken cMKR Mint 1000e18)
```

Reply:
Re-entered? That's weird... never seen or experienced that before. Sometimes if my /tmp/ directory is too full, saddle/repl have weird problems which seem to fix themselves after I reboot my computer.

Maybe this is the case here?

Reply:
This looks suspiciously like the old ganache fork bug